ev3dev/core.py

```diff
@@ -909,6 +909,8 @@ def wait(self, cond, timeout=None):
             if cond(self.state):
                 return True
 
+    def wait_until_not_moving(self, timeout=None):
```

Comment:
I used it when solving a 5x5x5 cube which was about 9 minutes of off/on movement of three large motors and a medium motor. If it had exited early it would have caused the cube to jam up but it was smooth sailing so I think it is good to go. 

I'll add a docstring to it in the morning.